dicodile/_dicodile.py

```diff
@@ -29,16 +98,16 @@ def dicodile(X, D_hat, reg=.1, z_positive=True, n_iter=100, strategy='greedy',
     params = DEFAULT_DICOD_KWARGS.copy()
     params.update(dicod_kwargs)
     params.update(dict(
-        strategy=strategy, n_seg=n_seg, z_positive=z_positive, tol=tol,
+        z_positive=z_positive, tol=tol,
         random_state=random_state, reg=reg_, timing=False, soft_lock='border',
         return_ztz=False, freeze_support=False, warm_start=True, debug=False
```

Comment:
`soft_lock='border'`, and `debug=False` should be set in the `DEFAULT_DICOD_KWARGS` to be overwrittable.
```